docs/agile-methodologies.md

```diff
@@ -28,4 +29,137 @@ It consists of the elaboration of a table or diagram in which three columns of t
+The sprint demo shouldn’t take up too much of a Scrum team’s time. Ensuring that the sprint review meeting is an informal affair where questions, feedback and discussion are welcome – allows for prep time to be kept to a minimum.
+
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/Njxek3s3Shs" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Demo steps
+
+1. **Tell a story:** This is one of the most important factors for a great demo, and also the most overlooked. Given the structured nature of Agile stories and epics, it’s easy to fall into the trap of simply enumerating the work that you’ve done. This isn’t necessarily bad, but it’s unlikely to excite non-technical stakeholders.
+2. **Let developers brag:** Whenever possible, it’s great to allow developers to present their own work, which helps to build confidence, morale, and a sense of ownership. A good compromise can be to have one organizing speaker with a different “guest” developer showing off their work each week.
+3. **Set expectations:** Setting expectations and providing context are critical for a successful demo. 
+4. **Action items:** List all completed stories in your agenda, weed out any stories that shouldn’t be demoed, organize the remaining stories roughly into scenarios or topics, decide whether to have developers help give parts of the demo and always set expectations and give context throughout the demo.
+
+### **Stand-up Session**
+
+A daily stand-up meeting is an opportunity for the project team to discuss a project’s progress at a high level. These meetings last 15 minutes and allow each contributor to report on their accomplishments since the last stand-up meeting.
+
+True to its name, all participants in stand-ups usually remain standing to keep the meetings short and on-topic. However, digital stand-ups are also possible. 
+
+In Agile project management, daily stand-up meetings are essential. These meetings allow project members to share critical information, openly discuss issues, and hold themselves and each other accountable.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/YHMw3CL5cXE" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Benefits of daily Agile stand-ups
+
+- Daily stand-ups allow team members to work collaboratively toward project goals. 
+- Daily stand-up meetings are important for keeping Agile teams focused and on-task while providing quick, project-level updates to the rest of the team. 
+- The Agile methodology is all about versatility and flexibility, it’s important to make tweaks and improvements to your meetings to fit your team’s needs. 
+- Your daily stand-up should inform and draw out issues so that you can keep your project on track and get ahead of issues before they pop up.
+
+
+## Modern Agile
```

Comment:
@steph222  lets move the modern agile section to the begining of the document

Reply:
Of course!

Reply:
Above the title  What are agile methodologies?